crates/nostr/Cargo.toml

```diff
@@ -15,13 +15,14 @@ keywords = ["nostr", "protocol", "sdk"]
 default = ["all-nips"]
 blocking = ["reqwest?/blocking"]
 vanity = ["nip19"]
-all-nips = ["nip04", "nip05", "nip06", "nip11", "nip19", "nip46"]
+all-nips = ["nip04", "nip05", "nip06", "nip11", "nip19", "nip40", "nip46"]
 nip03 = ["dep:nostr-ots"]
 nip04 = ["dep:aes", "dep:base64", "dep:cbc"]
 nip05 = ["dep:reqwest"]
 nip06 = ["dep:bip39", "dep:bitcoin"]
 nip11 = ["dep:reqwest"]
 nip19 = ["dep:bech32"]
+nip40 = []
```

Comment:
Since you are not adding another dependency, you can avoid to add this feature :)